### PR TITLE
Fix broken extend links

### PR DESCRIPTION
**Pull request by ethanshar**
## Description
Fix docs links by adding missing category in "extendLink"

## Changelog
Fix docs broken links

### CODE REVIEW

**Comment by ethanshar** [comment]
@Inbal-Tish 
Notice that some link were still broken. 
It usually tell you that when you try to deploy the docs (`npm run docs:deploy`)  and you can test it by running the docs from `docuilib` folder

**Comment by Inbal-Tish** [comment]
> @Inbal-Tish Notice that some link were still broken. It usually tell you that when you try to deploy the docs (`npm run docs:deploy`) and you can test it by running the docs from `docuilib` folder

I don't understand why add the category on these components to make the link work and not in all of the components?

**Comment by ethanshar** [comment]
> > @Inbal-Tish Notice that some link were still broken. It usually tell you that when you try to deploy the docs (`npm run docs:deploy`) and you can test it by running the docs from `docuilib` folder
> 
> I don't understand why add the category on these components to make the link work and not in all of the components?

What do you mean "all of the components"? 
Where for example you'd expect it to be?

**Comment by Inbal-Tish** [comment]
> > > @Inbal-Tish Notice that some link were still broken. It usually tell you that when you try to deploy the docs (`npm run docs:deploy`) and you can test it by running the docs from `docuilib` folder
> > 
> > 
> > I don't understand why add the category on these components to make the link work and not in all of the components?
> 
> What do you mean "all of the components"? Where for example you'd expect it to be?

I don't expect it to be used at all. What I'm asking is, take for example the Card component, we have the "extends" key with "ThouchableOpacity" so why do we need to specify the category, like ""basic/TouchableOpacity", in the Button component ?

**Comment by ethanshar** [comment]
> I don't expect it to be used at all. What I'm asking is, take for example the Avatar component, we have the "extends" key with "ThouchableOpacity" so why do we need to specify the category, like ""basic/TouchableOpacity", in the Button component ?

Yes, you are right, there are some inconsistencies.. i'll check it out

File: scripts/buildDocs.js [hunk 1]
```diff
@@ -110,3 +109,9 @@ function getComponentNameParts(componentName) {
     return [parts[1], parts[0]];
   }
 }
+
+function generateExtendsLink(extendsLink) {
+  const extendedComponentName = _.last(_.split(extendsLink, '/')); // Incubator/TextField -> TextField
+  const extendsText = `[${extendedComponentName}](/docs/components/${extendsLink})`;
```
[the remark this answers is not in the document]
Comment (ethanshar) [reply scripts/buildDocs.js:115]: The `component.category` is the not necessarily the category of the component it extends.

[the remark this answers is not in the document]
Comment (Inbal-Tish) [reply scripts/buildDocs.js:115]: Right... Ok. I guess there's no other way than...

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**